app/src/main/java/com/cornellappdev/android/eatery/WeeklyMenuFragment.java

```diff
@@ -0,0 +1,364 @@
+		DisplayMetrics dm = new DisplayMetrics();
+		getActivity().getWindowManager().getDefaultDisplay().getMetrics(dm);
+		int width = dm.widthPixels;
+		mExpListViewWest.setIndicatorBounds(width - 250, width);
+		mExpListViewNorth.setIndicatorBounds(width - 250, width);
+		mExpListViewCentral.setIndicatorBounds(width - 250, width);
+
+		mLastExpandedPosition = -1;
+		mLastClickedListView = null;
+
+		mExpListViewCentral.setOnGroupExpandListener(
+				new NonScrollExpandableListView.OnGroupExpandListener() {
+					@Override
+					public void onGroupExpand(int i) {
+						if (mLastClickedListView != null
+								&& mLastExpandedPosition != -1
+								&& i != mLastExpandedPosition) {
+							mLastClickedListView.collapseGroup(mLastExpandedPosition);
+						}
+						mLastExpandedPosition = i;
+						mLastClickedListView = mExpListViewCentral;
+					}
+				});
+
+		mExpListViewWest.setOnGroupExpandListener(
+				new NonScrollExpandableListView.OnGroupExpandListener() {
+					@Override
+					public void onGroupExpand(int i) {
+						if (mLastClickedListView != null
+								&& mLastExpandedPosition != -1
+								&& i != mLastExpandedPosition) {
+							mLastClickedListView.collapseGroup(mLastExpandedPosition);
+						}
+						mLastExpandedPosition = i;
+						mLastClickedListView = mExpListViewWest;
+					}
+				});
+		mExpListViewNorth.setOnGroupExpandListener(
+				new NonScrollExpandableListView.OnGroupExpandListener() {
+					@Override
+					public void onGroupExpand(int i) {
+						if (mLastClickedListView != null
+								&& mLastExpandedPosition != -1
+								&& i != mLastExpandedPosition) {
+							mLastClickedListView.collapseGroup(mLastExpandedPosition);
+						}
+						mLastExpandedPosition = i;
+						mLastClickedListView = mExpListViewNorth;
+					}
+				});
+
+		// Populate list of date TextViews on header
+		mDateList.add((TextView) rootView.findViewById(R.id.date0));
+		mDateList.add((TextView) rootView.findViewById(R.id.date1));
+		mDateList.add((TextView) rootView.findViewById(R.id.date2));
+		mDateList.add((TextView) rootView.findViewById(R.id.date3));
+		mDateList.add((TextView) rootView.findViewById(R.id.date4));
+		mDateList.add((TextView) rootView.findViewById(R.id.date5));
+		mDateList.add((TextView) rootView.findViewById(R.id.date6));
+
+
+		// When changing date, highlight Breakfast textview
+		mLinDate.setOnClickListener(
+				new View.OnClickListener() {
+					@Override
+					public void onClick(View view) {
+						mBreakfastText.setTextColor(ContextCompat.getColor(getActivity().getApplicationContext(), R.color.blue));
+						mLunchText.setTextColor(ContextCompat.getColor(getActivity().getApplicationContext(), R.color.secondary));
+						mDinnerText.setTextColor(ContextCompat.getColor(getActivity().getApplicationContext(), R.color.secondary));
+					}
+				});
+
+		Date now = new Date();
+		Calendar cal = Calendar.getInstance();
+		cal.setTime(now);
+		for (int i = 0; i < 7; i++) {
+			// Formatting for each day
```

Comment:
indentation is still 8 on GitHub, did you double check your linting settings?

Reply:
i'm planning on pushing linting changes at the very end!